cluster/mesos/docker/dind/build.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# Copyright 2015 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Builds a docker image that contains the kubernetes-mesos binaries.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+IMAGE_REPO=${IMAGE_REPO:-local/kubernetes-mesos-slave-dind}
+IMAGE_TAG=${IMAGE_TAG:-latest}
+
+script_dir=$(cd $(dirname "${BASH_SOURCE}") && pwd -P)
+KUBE_ROOT=$(cd ${script_dir}/../../../.. && pwd -P)
+
+# download nsenter
+mkdir -p "${script_dir}/downloads"
+docker run --rm -v "${script_dir}/downloads:/target" jpetazzo/nsenter
```

Comment:
This install process using docker makes alarms go off in my head. What is this doing and why can't we just do it in the Dockerfile?

The nsenter readme says "nsenter still needs to run from the host; it cannot run inside a container (yet)." Have we disproven that for our dind case?